readthedocs/doc_builder/exceptions.py

```diff
@@ -52,6 +52,13 @@ class BuildUserError(BuildBaseException):
         "and it is not currently supported. "
         'Please, remove all the files but the "{artifact_type}" your want to upload.'
     )
+    BUILD_OUTPUT_OLD_DIRECTORY_USED = gettext_noop(
+        "Your project is outputting files to an old directory ('_build/html') "
+        "when building the docs. "
+        "Note this directory is not used anymore and the result may be unexpected. "
+        "Please, check your build process looking for this directory and replace it by "
+        "the path '$READTHEDOCS_OUTPUT/html' instead."
+    )
```

Comment:
Hi @humitos :wave: 

This will be a great help for anyone who is stuck here!

```suggestion
    BUILD_OUTPUT_OLD_DIRECTORY_USED = gettext_noop(
        "Your project is writing files to an unsupported directory ('_build/html')."
        "This directory is not used anymore and as a result, your documentation "
        "website will not be found and uploaded."
        "Please check your build commands and use "
        "the path '$READTHEDOCS_OUTPUT/html' instead of '_build/html'."
    )
```

What do you think about this? I was wondering that the result isn't merely "unexpected" but actually expected? :) I.e. the entire site won't be found and uploaded?